modules/kmm-deploying-modules.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-deploying-modules_{context}"]
+= Deploying kernel modules
+
+For each `Module` resource, Kernel Module Management (KMM) can create a number of `DaemonSet` resources:
+
+* One `ModuleLoader` `DaemonSet` per compatible kernel version running in the cluster.
```

Comment:
I would not use a monospaced font for ModuleLoader as it's not an OpenShift object type, but rather one component of the solution. 
```suggestion
* One ModuleLoader `DaemonSet` per compatible kernel version running in the cluster.
```

Reply:
Done.